src/go/k8s/apis/redpanda/v1alpha1/cluster_types.go

```diff
@@ -269,6 +271,16 @@ type AdminAPITLS struct {
 	RequireClientAuth bool `json:"requireClientAuth,omitempty"`
 }
 
+// PandaproxyAPITLS configures the TLS of the Pandaproxy API
+//
+// If Enabled the Pandaproxy is configured to use TLS. For usage simplicity,
```

Comment:
please update the comment here, I think this is no longer true (about the kafka api)

Reply:
Thanks, updated with on secret.